src/data/README.md

```diff
@@ -18,7 +18,235 @@ TODO - describe the properties used in DevDot
+
+</details>
+
+#### `blocks`
+
+Each item in the `blocks` array represents a component on the page. Each of these items must have a `type` property, which can be one of the types listed below.
+
+<!-- block: type heading -->
+
+<details>
+<summary>Block type: <code>"heading"</code></summary>
+
+Heading blocks render HTML heading elements. Each block accepts the following properties:
+
+| Property | Type | Details |
+| --- | --- | --- |
+| `type` | `"heading"` | Block type |
+| `heading` | `string` | Text for the heading |
+| `level` | `2 \| 3 \| 4 \| 5 \| 6` | Semantic heading level, for example `2` becomes `<h2>`. Note that there is already an `<h1>` rendered for the page, so only values  should be used. |
+| `size` | `100 \| 200 \| 300 \| 400 \| 500` | Visual size of the heading. `500` is the largest size and `100` is the smallest. Visual size should generally reflect the semantic level, with `h2 = 300`, `h3 = 200`, and `h4` and below at the `100` size.
+
+Example: `h2` with `300` size:
+
+```json5
+{
+  "type": "heading",
+  "heading": "Featured Reference Docs",
+  "level": 2,
+  "size": 300,
+}
+```
+
+![](https://user-images.githubusercontent.com/4624598/158818745-a20d1892-efa6-4053-9d00-811645d642aa.png)
+
+Example: `h2` with `400` size:
+
+```json5
+{
+  "type": "heading",
+  "heading": "Explore Waypoint Documentation",
+  "level": 2,
+  "size": 400,
+}
+```
+
+![](https://user-images.githubusercontent.com/4624598/158818617-2b8ce029-ad41-4081-8701-869d51abf40b.png)
+
+</details>
+
+<!-- block type: get_started -->
+
+<details>
+<summary>Block type: <code>"get_started"</code></summary>
+
+The `"get_started"` block renders a heading, descriptive text, and a single link alongside a product icon.
+
+| Property | Type | Details |
+| --- | --- | --- |
+| `type` | `"get_started"` | Block type |
+| `product` | `"boundary" \| "consul" \| "nomad" \| "packer" \| "terraform" \| "vault" \| "vagrant" \| "waypoint" \| "sentinel" \| "hcp"` | Product icon to be shown. |
+| `heading` | `string` | Text for the heading |
+| `text` | `string` | Descriptive text shown below the heading |
+| `link` | `{ text: string, url: string }` | Link shown below the body text |
+
+Example: Waypoint `"get_started"` block
+
+```json5
+{
+  "type": "get_started",
+  "product": "waypoint",
+  "heading": "Introduction to Waypoint",
+  "text": "Welcome to Waypoint! This introduction section covers what Waypoint is, the problem Waypoint aims to solve, and how Waypoint compares to other software.",
+  "link": {
+    "text": "Get Started",
+    "url": "/waypoint/docs/intro"
+  }
+}
+```
+
+![](https://user-images.githubusercontent.com/4624598/158821262-03798dca-12e6-487b-ac3e-e8bab51be8b1.png)
+
+</details>
+
+<!-- block type: cards -->
+
+<details>
+<summary>Block type: <code>"cards"</code></summary>
+
+The `"cards"` block displays a grid of cards, each linked using a single `url`.
```

Comment:
💭 Similar thoughts here as previous comment, do you think it would be valuable to specify `CardLink` here?

```suggestion
The `"cards"` block displays a grid of `CardLink`s, each linked using a single `url`.
```

Reply:
I dig it 👍